### PR TITLE
Closes #24: Comments in integration-test-default.properties

### LINKED ISSUE #24: Comments in application/integration-test-default.properties
The properties in these 2 files should be described.

### PR DESCRIPTION
**Pull request by madryk**

### CODE REVIEW

**Comment by madryk** [comment]
When I was writing comments, I realized that properties `iis.hadoop.frontend.home.dir` and `iis.hadoop.frontend.user.dir` probably should be merged into one.
We can talk about it in our next meeting.

**Comment by lukdumi** [comment]
OK, this can be a good idea.

File: iis-core/src/test/resources/integration-test-default.properties [hunk 1]
```diff
@@ -1,13 +1,30 @@
...
+# of a gateway between cluster components and machines
+# that are not part of hadoop cluster.
+# It is required that frontend machine will provide ssh access
+
+# your username on frontend machine (must also match your username on hadoop cluster)
 iis.hadoop.frontend.user.name=change.to.username.on.server
+# path to user directory on frontend machine (relative to {iis.hadoop.frontend.home.dir})
+# user must have read/write rights to this directory
 iis.hadoop.frontend.user.dir=/change/to/user/dir
+# common for all users part of path to user directory
```
Comment (lukdumi) [inline iis-core/src/test/resources/integration-test-default.properties]: part of path to user directory, common for all users

File: iis-core/src/test/resources/integration-test-default.properties [hunk 2]
```diff
@@ -1,13 +1,30 @@
...
 iis.hadoop.frontend.home.dir=/tmp
+# host name of frontend machine
```
Comment (lukdumi) [inline iis-core/src/test/resources/integration-test-default.properties]: host name of <b>the</b> frontend machine

File: iis-core/src/test/resources/integration-test-default.properties [hunk 3]
```diff
@@ -1,13 +1,30 @@
...
 iis.hadoop.frontend.host.name=localhost
+# ssh port number for frontend machine
```
Comment (lukdumi) [inline iis-core/src/test/resources/integration-test-default.properties]: ... of the ...

File: iis-core/src/test/resources/integration-test-default.properties [hunk 4]
```diff
@@ -1,13 +1,30 @@
...
 iis.hadoop.frontend.port.ssh=22
 
+# location of oozie service (as seen by frontend machine) 
 oozieServiceLoc=http://localhost:11000/oozie
+# address to name node (as seen by frontend machine)
```
Comment (lukdumi) [inline iis-core/src/test/resources/integration-test-default.properties]: address of the name node (as ... the frontend...)

File: iis-core/src/test/resources/integration-test-default.properties (hunk 1 above)
Comment (madryk) [inline iis-core/src/test/resources/integration-test-default.properties]: ok

File: iis-core/src/test/resources/integration-test-default.properties (hunk 2 above)
Comment (madryk) [inline iis-core/src/test/resources/integration-test-default.properties]: :+1:

File: iis-core/src/test/resources/integration-test-default.properties (hunk 3 above)
Comment (madryk) [inline iis-core/src/test/resources/integration-test-default.properties]: :+1:

File: iis-core/src/test/resources/integration-test-default.properties (hunk 4 above)
Comment (madryk) [inline iis-core/src/test/resources/integration-test-default.properties]: ok

**Pull request closed without merging**